generator/constants.ts

```diff
@@ -29,8 +29,8 @@ export const managementGroupRootSchema = {
 };
 export const generatedSchemasTemplatePath = path.join(__dirname, 'resources/autogeneratedResources_template.json');
 
-export const autorestCoreVersion = '3.0.6274';
-export const azureresourceschemaVersion = '3.0.79';
+export const autorestCoreVersion = '3.0.6320';
+export const azureresourceschemaVersion = '3.0.92';
```

Comment:
Is this needed for this PR? If not, possible to submit this as a separate change?

Reply:
I will regenerate the PR, since we start use `--multiapi` to generate schemas